app/sidekiq/update_statistics_caches_job.rb

```diff
@@ -0,0 +1,30 @@
+class UpdateStatisticsCachesJob
+  include Sidekiq::Job
+
+  BATCH_SIZE = 50
+
+  def perform
+    links.each_slice(BATCH_SIZE) do |slice|
```

Comment:
would it be worth adding some logs here? I think otherwise jobs happen really silently and it's hard to follow them

Reply:
Good plan!